ext/opentelemetry-ext-dbapi/src/opentelemetry/ext/dbapi/__init__.py

```diff
@@ -125,6 +125,15 @@ def wrap_connect_(
         logger.warning("Failed to integrate with DB API. %s", str(ex))
 
 
+def unwrap_connect(
+    connect_module: typing.Callable[..., any], connect_method_name: str,
+):
+    if hasattr(connect_module, connect_method_name):
+        conn = getattr(connect_module, connect_method_name)
+        if isinstance(conn, wrapt.ObjectProxy):
+            setattr(connect_module, connect_method_name, conn.__wrapped__)
+
+
 class DatabaseApiIntegration:
```

Comment:
Any reason to use BaseInstrumentor in here as well?, people could be using dbapi instrumentor directly

Reply:
`BaseInstrumentor` is used to implement an interface that allows to instrument all the calls made to a given library, for instance pymysql. AFAIU dbapi is a specification and not a library, so having an instrumentor here doesn't make sense. If a person wants to use dpapi directly, they should use the `wrap_connect` and `unwrap_connect` functions.

Reply:
Should document this? In the docstrings `trace_integration()` is the recommended way to instrument. Perhaps we should expose an API method to uninstrument (since there isn't really a semantic similarity between `trace_integration` and `unwrap_connect`.

Reply:
The PyMySQL docstrings was wrong, `trace_integration` doesn't exist anymore. 
`PymysqlInstrumentor` offers two methods, `instrument` and `uninstrument`. 

I'm not sure what to do with the `trace_integration` on dpapi, it's very similar to `wrap_connect` but creates a tracer before. Maybe we could remove it and let the user play directly with `wrap_connect`...